fjord/suggest/providers/trigger/provider.py

```diff
@@ -0,0 +1,112 @@
+import logging
+
+from fjord.base.google_utils import ga_track_event
+from fjord.redirector import (
+    Redirector,
+    RedirectParseError,
+    build_redirect_url
+)
+from fjord.suggest import Link, Suggester
+from fjord.suggest.providers.trigger.models import TriggerRule
+
+
+PROVIDER = 'trigger'
+PROVIDER_VERSION = 1
+
+TRIGGER = 'tgr'
+EVENT_CATEGORY = 'trigger' + str(PROVIDER_VERSION)
+SESSION_KEY = 'trigger_{0}'
+
+
+logger = logging.getLogger('i.trigger')
+
+
+def format_redirect(slug):
+    return '{0}.{1}'.format(TRIGGER, slug)
+
+
+def parse_redirect(redirect):
+    try:
+        trigger, slug = redirect.split('.')
+        return trigger, slug
```

Comment:
Actually, I take it back. Having the binding first makes it assert that there are exactly 2 parts, which is useful.